google-cloud-sdk/testSrc/com/google/cloud/tools/intellij/appengine/java/sdk/ManagedCloudSdkUpdateServiceTest.java

```diff
@@ -110,7 +111,10 @@ public void update_scheduledNow_ifLastUpdate_elapsed() {
     managedCloudSdkUpdateService.activate();
 
     verify(managedCloudSdkUpdateService)
-        .schedule(any(), eq(0L), eq(ManagedCloudSdkUpdateService.SDK_UPDATE_INTERVAL_MS));
+        .schedule(
+            any(),
+            eq(0L),
+            ArgumentMatchers.eq(ManagedCloudSdkUpdateService.SDK_UPDATE_INTERVAL_MS));
```

Comment:
why did you stop using the static import on eq (which is still imported)? Same for some of the other tests

Reply:
No idea, I think I ran `spotlessApply` and it did this for some reason. Reverted, thanks.